packages/slice-machine/src/features/customTypes/actions/renameCustomType.ts

```diff
@@ -0,0 +1,46 @@
+import { CustomType } from "@prismicio/types-internal/lib/customtypes";
+import { managerClient } from "@src/managerClient";
+import { CUSTOM_TYPES_MESSAGES } from "../customTypesMessages";
+import { CustomTypeFormat } from "@slicemachine/manager";
+import { toast } from "react-toastify";
+
+type RenameCustomTypeArgs = {
+  model: CustomType;
+  newName: string;
+  onSuccess: (renamedCustomType: CustomType) => void;
+};
+export async function renameCustomType({
+  model,
+  newName,
```

Comment:
You added "newLabel" in another place and newName here, short thing but I feel better to have one naming "newLabel", WDYT?